src/pages/GetInvolved/Club.js

```diff
@@ -0,0 +1,112 @@
+import React from "react";
+import { useSelector } from "react-redux";
+import { withRouter, Link } from "react-router-dom";
+import Layout from "components/Layout/Layout";
+
+const Club = ({ match }) => {
+  let clubDataAll = useSelector((state) => state.clubs);
+  let currentClub = `${match.params.clubId}`;
+  let clubData = useSelector((state) =>
+    clubDataAll.find((object) => object.slug === currentClub)
+  );
+
+  if (!clubData) {
+    return null;
+  } else {
+    const createMarkup = () => {
+      return { __html: clubData.acf.club.club_description };
+    };
+    return (
+      <Layout>
+        <div className="club__container">
+          <p className="club__nav">
+            <Link className="club__nav--item" to="/">
+              Home
+            </Link>{" "}
+            /{" "}
+            <Link className="club__nav--item" to="/clubs">
+              Join a HUOA Club
+            </Link>{" "}
+            / {clubData.acf.seo.title}
+          </p>
+          <br />
+          <div className="club__header">
+            {!clubData.acf.club.club_image ? null : (
+              <img
+                className="club__header--image"
+                src={`${clubData.acf.club.club_image}`}
```

Comment:
These should be strings in react already

```suggestion
                src={ clubData.acf.club.club_image }
```